ZipBuilder/Sources/ManifestReader/FirebaseSDKs.pb.swift

```diff
@@ -59,7 +59,7 @@ public struct ZipBuilder_SDK {
   }
 
   /// MPM name for the blueprint.
-  var mpmName: String {
+  public var mpmName: String {
```

Comment:
If I remember right, I got warnings for these `public`s. Is there a reason they're added back?

Reply:
I needed access to the `openSource` variable but for whatever previous reason it was private. Nothing was specified in the `.proto` itself. I didn't see any warnings, so I'm not sure what it could have been previously. Maybe this was before it was split up into a separate module?

I can manually remove the `public` designations for the variables we're not using, but I haven't been able to see any warnings yet in any of the targets.

Reply:
No need. I may be misremembering what it was ....

Reply:
I'll fix it if anything comes up!